pam/internal/adapter/userselection.go

```diff
@@ -158,3 +158,11 @@ func (m *userSelectionModel) Focus() tea.Cmd {
 	m.selected = false
 	return m.Model.Focus()
 }
+
+// View renders a text view of the user selection UI.
+func (m userSelectionModel) View() string {
+	if !m.enabled {
+		return ""
+	}
+	return m.Model.View()
+}
```

Comment:
I love it when the description is 3 times bigger than the change :joy: 

When I was reading the PR description, I was expecting way more diff but I'm glad it was a simple issue to fix!